networkx/algorithms/distance_measures.py

```diff
@@ -125,32 +128,31 @@ def extrema_bounding(G, compute="diameter"):
                 for i in candidates
                 if ecc_upper[i] <= maxlower and 2 * ecc_lower[i] >= maxupper
             }
-
         elif compute == "radius":
             ruled_out = {
                 i
                 for i in candidates
                 if ecc_lower[i] >= minupper and ecc_upper[i] + 1 <= 2 * minlower
             }
-
         elif compute == "periphery":
             ruled_out = {
                 i
                 for i in candidates
                 if ecc_upper[i] < maxlower
                 and (maxlower == maxupper or ecc_lower[i] > maxupper)
             }
-
         elif compute == "center":
             ruled_out = {
                 i
                 for i in candidates
                 if ecc_lower[i] > minupper
                 and (minlower == minupper or ecc_upper[i] + 1 < 2 * minlower)
             }
-
         elif compute == "eccentricities":
-            ruled_out = {}
+            ruled_out = set()
+        else:
+            msg = "The argument passed to compute parameter is invalid. Please enter one of the following extreme distance metrics: 'diameter', 'radius', 'periphery', 'center', 'eccentricities'"
+            raise nx.NetworkXError(msg)
```

Comment:
```suggestion
            raise ValueError(msg)
```
I think a `ValueError` would be more appropriate here.